src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -596,6 +596,12 @@ void ProtocolHandlerImpl::SendEndServicePrivate(int32_t primary_connection_id,
                                                 uint8_t session_id,
                                                 uint8_t service_type) {
   LOG4CXX_AUTO_TRACE(logger_);
+  if (0 == connection_id) {
+    LOG4CXX_WARN(logger_,
```

Comment:
@Ypostolov Please make constant variable for more understanding same as in this [pull request](https://github.ford.com/SYNC/sdl_core/pull/357/files)